mne/surface.py

```diff
@@ -1667,19 +1667,27 @@ def marching_cubes(image, level, smooth=0):
     ----------
     image : ndarray
         The image to compute marching cubes with.
-    level : float
+    level : float | int | list
```

Comment:
It might simplify things to require a list and then just return a list of tuples, especially if it's a private function that would make sense